### PR TITLE
updated the last contacted time to be always there and be real

### PR DESCRIPTION
**Pull request by jimitpatel9**

### CODE REVIEW

**Comment by jackofseattle** [comment]
does this work when we use date strings? if so, LGTM

**Comment by jimitpatel9** [comment]
Are we using date strings??

File: client/vivace/list/template.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
                 <p flex>
                 <div layout="row" layout-align="end center" class="md-caption">
-                  Last contacted {{user.lastContact }}
+                  Last contacted {{user.lastContact | date :'fullDate'}}
```
Comment (prashantpawar) [inline client/vivace/list/template.html]: Why is it called fulldate? I don't see this value being used anywhere.

Comment (jimitpatel9) [inline client/vivace/list/template.html]: Its a filter which will show full date like :- Friday 22 December 2015

Comment (prashantpawar) [inline client/vivace/list/template.html]: Gotcha

**Comment by prashantpawar** [comment]
LGTM, merging

File: client/vivace/list/template.html (hunk 1 above)
Comment (jimitpatel9) [inline client/vivace/list/template.html]: we can also make it :- MM-DD-YYYY 
I feel like it would be easy to read if we have 'fuldate' format

**Pull request merged**